syntax = "proto3";

message SubdivisionEvent {
    enum Action {
        CREATED = 0;
        UPDATED = 1;
        DELETED = 2;
        LICENSE_ADDED = 3;
        LICENSE_EXPIRED = 4;
    }
    string type_name = 1;
    string name = 2;
    string location = 3;
    string tenant_id = 4;
    string link_to_subdivision_processing_domain = 5;
    string work_status = 6;
    string id = 7;
    Action action = 8;
}